### PR TITLE
Inventory creativedelete: German translation

### PR DESCRIPTION
**Pull request by ORelio**
Hi, some lines need translation to German before merging. @cyb3rko @Daenges could you please look into this and propose changes in this PR? Thanks 😉

### CODE REVIEW

**Comment by ORelio** [comment]
Affected lines:
```ini
cmd.inventory.creative_delete=Requested to clear slot #{0}
cmd.inventory.creative_fail=Failed to request Creative action
cmd.inventory.help.creativegive=Give item in creative mode.
cmd.inventory.help.creativedelete=Clear slot in creative mode.
```

File: MinecraftClient/Resources/lang/de.ini [hunk 1]
```diff
@@ -254,7 +254,8 @@ cmd.health.response=Gesundheit: {0}, Sättigung: {1}, Level: {2}, TotalExperienc
 # Inventory
 cmd.inventory.desc=Inventar-Befehl
 cmd.inventory.creative_done={0} x{1} im Slot #{2} angefordert.
-cmd.inventory.creative_fail=Item-Vergabe durch den Creative Modus wurde abgelehnt.
+cmd.inventory.creative_delete=Requested to clear slot #{0}
```
Comment (Daenges) [inline MinecraftClient/Resources/lang/de.ini:257]:
```suggestion
cmd.inventory.creative_delete=Anfrage zum Leeren des Slots #{0} gesendet.
```

File: MinecraftClient/Resources/lang/de.ini [hunk 2]
```diff
@@ -274,12 +275,14 @@ cmd.inventory.drop_stack=Droppe ganzen Item Stack aus Slot #{0}
...
 cmd.inventory.help.basic=Einfache Bedienung
 cmd.inventory.help.available=Verfügbare Aktionen
-cmd.inventory.help.help=\n{0} Benutze '/inventory help <action>' für weitere Hilfe.\nCreative mode give: {1}\n'player' und 'container' können mit 'p' und 'c' abgekürzt werden.\nParameter in '[]' sind optional.
+cmd.inventory.help.help={0}\nBenutze '/inventory help <action>' für weitere Hilfe.\n'player' und 'container' können mit 'p' und 'c' abgekürzt werden.\nParameter in '[]' sind optional.
 cmd.inventory.help.usage=Benutzung
 cmd.inventory.help.list=Liste deine Inventare auf.
 cmd.inventory.help.close=Schließe einen offenen Container.
 cmd.inventory.help.click=Klicke auf ein Item.
 cmd.inventory.help.drop=Werfe ein Item aus deinem Inventar.
+cmd.inventory.help.creativegive=Give item in creative mode.
```
Comment (Daenges) [inline MinecraftClient/Resources/lang/de.ini:284]:
```suggestion
cmd.inventory.help.creativegive=Gebe dir ein Item im Kreativ-Modus
```

File: MinecraftClient/Resources/lang/de.ini [hunk 3]
```diff
@@ -274,12 +275,14 @@ cmd.inventory.drop_stack=Droppe ganzen Item Stack aus Slot #{0}
...
+cmd.inventory.help.creativedelete=Clear slot in creative mode.
```
Comment (Daenges) [inline MinecraftClient/Resources/lang/de.ini:285]:
```suggestion
cmd.inventory.help.creativedelete=Leere Slot im Kreativmodus.
```

**Review by Daenges — COMMENTED** [review]
Translating

File: MinecraftClient/Resources/lang/de.ini (hunk 1 above)
Comment (cyb3rko) [reply MinecraftClient/Resources/lang/de.ini:257]: Maybe remove the full stop

**Comment by ORelio** [comment]
Thanks @Daenges and @cyb3rko for your swift reply! Any suggestion for this line?
```ini
cmd.inventory.creative_fail=Failed to request Creative action
```

File: MinecraftClient/Resources/lang/de.ini [hunk 4]
```diff
@@ -254,7 +254,8 @@ cmd.health.response=Gesundheit: {0}, Sättigung: {1}, Level: {2}, TotalExperienc
 # Inventory
 cmd.inventory.desc=Inventar-Befehl
 cmd.inventory.creative_done={0} x{1} im Slot #{2} angefordert.
-cmd.inventory.creative_fail=Item-Vergabe durch den Creative Modus wurde abgelehnt.
+cmd.inventory.creative_delete=Anfrage zum Leeren des Slots #{0} gesendet
+cmd.inventory.creative_fail=Failed to request Creative action
```
Comment (cyb3rko) [inline MinecraftClient/Resources/lang/de.ini:258]:
```suggestion
cmd.inventory.creative_fail=Anfrage für Kreativ-Vorgang fehlgeschlagen
```

**Review by cyb3rko — CHANGES_REQUESTED** [review]

File: MinecraftClient/Resources/lang/de.ini (hunk 4 above)
Comment (cyb3rko) [reply MinecraftClient/Resources/lang/de.ini:258]: @Daenges What do you think about "Kreativ-Vorgang"? Any other idea?

Comment (Daenges) [reply MinecraftClient/Resources/lang/de.ini:258]: Change "Vorgang" to "Aktion".

**Comment by cyb3rko** [comment]
Should be fine now.

**Comment by ORelio** [comment]
Thanks @Daenges, @cyb3rko 👍

**Pull request merged**